lib/AssemblyAPI/Utils.py

```diff
@@ -23,50 +59,46 @@ def get_genome_annotations(workspace_url, ref):
 
 
 def get_external_source_info(workspace_url, ref):
-    return _get_data_from_ws(workspace_url, ref, ['external_source', 'external_source_id',
-                                                 'external_source_origination_date'])
+    included = ['external_source', 'external_source_id', 'external_source_origination_date']
+    return {k: _get_data_from_ws(workspace_url, ref, included)[k] for k in included}
```

Comment:
a little confused about this statement. Maybe switch from 
```
return {k: _get_data_from_ws(workspace_url, ref, included)[k] for k in included}
```
to
```
data = _get_data_from_ws(workspace_url, ref, included)
return {k: data[k] for k in included}
```
which is still a bit confusing to me,
not sure why this was changed.

Reply:
So the dict comprehension is needed because now _get_data_from_ws might return more than the requested attributes. However, your formulation makes much more sense